doc/generic/pgf/text-en/pgfmanual-en-tikz-matrices.tex

```diff
@@ -475,6 +475,53 @@ \subsubsection{Cell Styles and Options}
 \end{tikzpicture}
 \end{codeexample}
 
+In some cases, it is desirable to include some automation in each column/row
+separately. A typical example is to apply stripe-pattern to almost all columns
+and treat only a few of them differently. Then, nesting these keys can open up
+a lot of possibilities; in the following example a "feature comparison" table
+is demonstrated. It is intentionally made rather verbose to show how the column
+and row settings can be progressively overwritten to create certain effects.
+
+\begin{codeexample}[preamble={\usetikzlibrary{matrix,shapes.misc}}]
+\begin{tikzpicture}
+  [mycol/.style={column #1/.append style={
+     every even row/.style={nodes={fill=gray!10}}}
```

Comment:
I would change some colors so that they are not that bright/and or better distinguishable from `codeexample`s background color. So here I would change `gray!10` to `gray!20` ...